app/services/decidim/action_delegator/participants_csv_importer.rb

```diff
@@ -0,0 +1,208 @@
+# frozen_string_literal: true
+
+module Decidim
+  module ActionDelegator
+    class ParticipantsCsvImporter
+      include Decidim::FormFactory
+
+      def initialize(csv_file, current_user, current_setting)
+        @csv_file = csv_file
+        @current_user = current_user
+        @current_setting = current_setting
+      end
+
+      def import!
+        import_summary = {
+          total_rows: 0,
+          imported_rows: 0,
+          error_rows: [],
+          skipped_rows: [],
+          details_csv_path: nil
+        }
+
+        details_csv_file = File.join(File.dirname(@csv_file), "details.csv")
+
+        ActiveRecord::Base.transaction do
```

Comment:
Are we needing a transaction here?
I think if each row is imported one by one and some fails, it will provoke a full revocation if we use transactions.